components/rhel/rhel-ose/Vagrantfile

```diff
@@ -69,8 +76,19 @@ Vagrant.configure(2) do |config|
     sudo setsebool -P virt_sandbox_use_fusefs 1
   SHELL
 
+  # landrush
+  config.landrush.enabled = true
+  config.landrush.tld = TLD
+  config.landrush.host TLD, HOSTNAME
+  config.landrush.guest_redirect_dns = false
```

Comment:
@praveenkumar this line is important and will make this setup even work on Libvirt. Basically we skip the re-configuration of DNS on the guest. We are anyways only interested on the exposure on the host